<a href="https://colab.research.google.com/github/angelafrocha/estatistica/blob/main/Distribui%C3%A7%C3%A3o_binomial.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

Em um concurso para preencher uma vaga de cientista de dados temos um total de 10 questões de múltipla escolha com 3 alternativas possíveis em cada questão. Cada questão tem o mesmo valor. Suponha que um candidato resolva se aventurar sem ter estudado absolutamente nada. Ele resolve fazer a prova de olhos vendados e chutar todas as resposta. Assumindo que a prova vale 10 pontos e a nota de corte seja 5, obtenha a probabilidade deste candidato acertar 5 questões e também a probabilidade deste candidato passar para a próxima etapa do processo seletivo.

In [1]:
from scipy.stats import binom

In [5]:
# Probablilidade de acertar 5 questões
p = 1/3
n = 10
k = 5

probabilidade_5 = binom.pmf(5, n, p)
probabilidade_5

0.13656454808718146

In [7]:
# Probablilidade de passar
p = 1/3
n = 10
k = [5,6,7,8,9,10]

prob_passar = binom.pmf(k, n, p).sum()
prob_passar

0.21312808006909476

**A probabilidade de acertar 5 questões é de 0.13656454808718146**

**E a probablilidade de passar é de 0.21312808006909476**